### DISCUSSION TITLE
Stability has improved noticeably - thank you Metals team!

### DISCUSSION THREAD

**Discussion by LeUser111** [scalameta/metals#6951] [General]
Hi there,

I've been noticing for a while now that Metals is really improving in regards to stability. 

I used to have problems with disconnecting bloop sessions/timeouts or e.g. new test cases not being picked up unless the editor tab was closed and re-opened, code not being compiled from time to time unless the window lost and re-gained focus, ...

All of those issues seem to be pretty much resolved now. Developing with metals is currently super smooth and things are working really well on my end!

Thank you very much to all of the contributors!

**Comment by tgodzik** [comment]
Thanks for the kind words! There is still plenty to fix, so don't hesitate to report issues.
